src/pages/components/ai-label/code.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: AI label
+description: The AI label indicates an instance of AI in the UI.
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+<PageDescription>
+
+Preview the AI label component with the React live demo. For detailed code usage
+documentation, see the Storybooks for each framework below.
+
+</PageDescription>
+
+## Documentation
+
+<Row className="resource-card-group">
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="React"
+    href="https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-slug--overview"
```

Comment:
Just one note. We will have to revisit the resource cards once the name chages in storybook to ai-label